### DISCUSSION TITLE
Any way to "have a conversation" with CoPilot like ChatGPT?

### DISCUSSION THREAD

**Discussion by pkellner** [community/community#51615] [Copilot Conversations]
### Select Topic Area

Question

### Body

For example, I want to ask questions about TypeScript or even just feed it a snipped of code and ask to change in someway.

Like for example, I asked ChatGPT

"what is the simplest way to call useState in react with string when using typescript"

and then follow with

"is there anything simpler"?

Is there a way to do that with CoPilot? That is, have a conversation while I'm not in a code file. I'm currently using the integration of CoPilot with WebStorm and VS Code.

**Comment by bewuethr** [comment]
Sounds exactly like the context aware conversations announced for [Copilot X](https://github.com/features/preview/copilot-x); there's a [waitlist](https://github.com/github-copilot/chat_waitlist_signup/join) for the feature.

**Reply by bewuethr** [reply]
Update: [GitHub Copilot Chat](https://docs.github.com/en/copilot/github-copilot-chat/about-github-copilot-chat) is now a thing.

**Comment by pkellner** [comment]
Thanks @bewuethr . I guess I keep using ChatGPT for that.   $10/mo copilot, $20/mo ChatGPT, this could turn into real money soon.
